src/mlpack/methods/logistic_regression/logistic_regression_function.hpp

```diff
@@ -28,10 +28,25 @@ template<typename MatType = arma::mat>
 class LogisticRegressionFunction
 {
  public:
+  /**
+   * Creates the LogisticRegressionFunction.
+   *
+   * @param predictors X, matrix of data points.
+   * @param responses y, the measured data for each point in X.
+   * @param lambda Regularization constant for ridge regression.
+   */
   LogisticRegressionFunction(const MatType& predictors,
                              const arma::Row<size_t>& responses,
                              const double lambda = 0);
 
+  /**
+   * Creates the LogisticRegressionFunction with initialPoint.
+   *
+   * @param predictors X, matrix of data points.
+   * @param responses y, the measured data for each point in X.
```

Comment:
Same here as above.

Reply:
@ShikharJ  I did the changes suggested by you